LMStudioLLM class added. It may not be perfect but it is working. All…

AlonAshken opened the pull request:
…ows users to easily interact using LangChain with a hosted Local Inference Server in Meta's LMStudio. Feel free to corrent/add things to it. it is working well but may not be complete

Thank you for contributing to LangChain!

- [x] **PR title**: "package: lmstudio. It is a package to allow interactions with LMStudio hosted LLMs."



- [x] **PR message**:
    - **Description:** Added an LLM to allow interacting with LMStudio hosted LLMs.
    - **Dependencies:** Only depends on BaseLLM
    - **Twitter handle:** Don't mind much about that but here's my handle: @AlonAshkenazi69
.

AlonAshken committed:
LMStudioLLM class added. It may not be perfect but it is working. Allows users to easily interact using LangChain with a hosted Local Inference Server in Meta's LMStudio. Feel free to corrent/add things to it. it is working well but may not be complete

AlonAshken committed:
linting fixes

AlonAshken committed:
Merge pull request #1 from langchain-ai/master

Adding LMStudio local LLM support.

baskaryan committed:
Merge branch 'master' into lmstudio-llm

baskaryan reviewed (commented):
can we add a page to the docs at `docs/docs/integrations/llms/...`

AlonAshken commented:
> can we add a page to the docs at `docs/docs/integrations/llms/...`

Sure, I've added another piece for LMStudioEmbeddings and added docs for both of them.
